sample-code/pmd-example-rules/pom.xml

```diff
@@ -21,27 +21,10 @@ a JAR containing everything needed to register and run a custom rule.
 	</properties>
 
 	<dependencies>
-		<dependency>
-			<groupId>net.sourceforge.pmd</groupId>
-			<artifactId>pmd</artifactId>
-			<version>6.55.0</version>
-			<type>pom</type>
-		</dependency>
 		<dependency>
 			<groupId>net.sourceforge.pmd</groupId>
 			<artifactId>pmd-apex</artifactId>
```

Comment:
I just tested this locally and built it successfully. The only dependency needed is the pmd-apex v7.0.0 now.